### PR TITLE
docs: add French translation of guides/configure-biome.mdx

### PR DESCRIPTION
**Pull request by webdevbynight**
## Summary

Add the translation into French of the “Configure Biome” page.

Added:
- the `guides/configure-biome.mdx` file translated into French.

### CODE REVIEW

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 1]
```diff
@@ -0,0 +1,387 @@
+---
+title: Configurer Biome
+description: Apprenez comment la configuration de Biome fonctionne.
+---
+
+import {FileTree} from '@astrojs/starlight/components';
+
+Ce guide vous aidera à comprendre comment configurer Biome.
+Il explique la structure d’un fichier de configuration de Biome et la manière dont Biome résoud sa configuration.
+Si vous êtes déjà familiarisé avec la configuration, il se peut que vous vouliez jeter un coup d’œil à la [référence de la configuration](/fr/reference/configuration/), qui détaille toutes les options disponibles.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:10]:
```suggestion
Si vous êtes déjà familiarisé avec la configuration de Biome, il se peut que vous vouliez jeter un coup d’œil à la [référence de la configuration](/fr/reference/configuration/), qui détaille toutes les options disponibles.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+
+Biome vous permet de personnaliser son comportement en utilisant des options en ligne de commande ou un fichier de configuration appelé `biome.json` ou `biome.jsonc`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:12]:
```suggestion
Biome vous permet de personnaliser son comportement en utilisant des options en ligne de commande ou un fichier de configuration nommé `biome.json` ou `biome.jsonc`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+Nous vous recommandons de créer un fichier de configuration pour chaque projet,
+ce qui assure que chaque membre de l’équipe a la même configuration dans l’interface en ligne de commande et dans tout éditeur qui permet l’intégration de Biome.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:14]:
```suggestion
ce qui assure que chaque membre de votre équipe a la même configuration `la fois en ligne de commande et dans tout éditeur doté de l'extension de Biome.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+Beaucoup des options disponibles dans un fichier de configuration le sont également en ligne de commande.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:15]:
```suggestion
La plupart des options disponibles dans le fichier de configuration sont également disponibles en ligne de commande.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+
+## Structure du fichier de configuration
+
+Un fichier de configuration de Biome est appelé `biome.json` ou `biome.jsonc`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:19]:
```suggestion
Le fichier de configuration de Biome est nommé `biome.json` ou `biome.jsonc`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+Il est habituellement placé dans le répertoire racine de votre projet, à côté du `package.json` de ce dernier.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:20]:
```suggestion
Il est habituellement placé dans le répertoire racine de votre projet où se trouve gténéralement `package.json`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 7]
```diff
@@ -0,0 +1,387 @@
...
+
+Parce que Biome est un ensemble d’outils, sa configuration est organisée autour des outils qu’il fournit.
+En ce moment, Biome fournit trois outils&nbsp;: l‘outil de formatage, l’outil de linting et le trieur d’imports (aussi appelé l’organisateur d’imports).
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:23]:
```suggestion
Biome fournit actuellement trois outils&nbsp;: l‘outil de formatage, l’outil de linting et le trieur d’imports (aussi appelé l’organisateur d’imports).
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 8]
```diff
@@ -0,0 +1,387 @@
...
+Tous ces outils sont activés par défaut.
+Vous pouvez désactiver un ou plusieurs d’entre eux en utilisant le champ `<outil>.enabled`&nbsp;:
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:25]:
```suggestion
Vous pouvez désactiver un ou plusieurs d’entre eux en utilisant la propriété `<outil>.enabled`&nbsp;:
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 9]
```diff
@@ -0,0 +1,387 @@
...
+    "enabled": false
+  },
+  "organizeImports": {
+    "enabled": false
+  }
+}
+```
+
+Les options qui s’appliquent à plus d’un langage sont placées dans le champ de l’outil correspondant.
+Les options d’un outil propres à un langage sont placées sous un champ `<langage>.<outil>`,
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:43]:
```suggestion
Les options propres à un langage sont placées sous un champ `<langage>.<outil>`,
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 10]
```diff
@@ -0,0 +1,387 @@
...
+ce qui inclut TypeScript, JSX et TSX.
+:::
+
+
+## Résolution du fichier de configuration
+
+Biome utilise l’autodécouverte pour trouver le fichier de configuration le plus proche.
+Il regarde dans le répertoire de travail et dans les répertoires parents jusqu’à ce qu’il trouve un fichier `biome.json` ou `biome.jsonc`.
+Si aucune configuration n’est trouvée, alors il applique les réglages par défaut de Biome.
+Si `biome.json` et `biome.jsonc` sont présents en même temps dans le même dossier, priorité sera donnée à `biome.json`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:81]:
```suggestion
Si `biome.json` et `biome.jsonc` sont présents en même temps dans le même dossier, la priorité sera donnée à `biome.json`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 11]
```diff
@@ -0,0 +1,387 @@
...
+:::note
+Biome fait référence sous le nom de `javascript` à toutes les variantes du langage JavaScript,
+ce qui inclut TypeScript, JSX et TSX.
+:::
+
+
+## Résolution du fichier de configuration
+
+Biome utilise l’autodécouverte pour trouver le fichier de configuration le plus proche.
+Il regarde dans le répertoire de travail et dans les répertoires parents jusqu’à ce qu’il trouve un fichier `biome.json` ou `biome.jsonc`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:79]:
```suggestion
Biome prend en compte le fichier de configuration le plus proche en débutant sa recherche dans le répertoire de travail courrant et en remontant la hierarchie de répertoires jusqu'a trouver un fichier `biome.json` ou `biome.jsonc`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 12]
```diff
@@ -0,0 +1,387 @@
...
+      - package.json
+</FileTree>
+
+
+- Les commandes de Biome qui s’exécutent dans `app/backend/package.json` utiliseront le fichier de configuration `app/backend/biome.json`&nbsp;;
+- les commandes de Biome qui s’exécutent dans `app/frontend/legacy/package.json` et `app/frontend/new/package.json`
+utiliseront le fichier de configuration `app/frontend/biome.json`&nbsp;;
+
+:::caution
+Biome ne prend pas en charge les fichiers `biome.json` imbriqués, ni en ligne de commande ni en LSP. [Suivre et aider à la prise en charge dans le problème correspondant](https://github.com/biomejs/biome/issues/2228)
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:104]:
```suggestion
Biome ne prend pas en charge les fichiers `biome.json` imbriqués, ni en ligne de commande ni en LSP. [Suivre et aider à la prise en charge dans le ticket correspondant](https://github.com/biomejs/biome/issues/2228)
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 13]
```diff
@@ -0,0 +1,387 @@
...
+:::
+
+:::note
+Les commandes de Biome prennent en charge l’utilisation de l’option `--config-path` et de la variable d’environnement `BIOME_CONFIG_PATH`,
+ce qui vous permet de spécifier un fichier de configuration personnalisé ou un répertoire où Biome peut chercher un fichier `biome.json` ou `biome.jsonc`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:109]:
```suggestion
ce qui vous permet de spécifier un fichier de configuration personnalisé ou un répertoire où Biome peut trouver un fichier `biome.json` ou `biome.jsonc`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 14]
```diff
@@ -0,0 +1,387 @@
...
+Voici un exemple de la manière dont vous pourriez mettre en place votre configuration pour étendre un fichier de configuration `common.json`&nbsp;:
+
+```json title="biome.json"
+{
+  "extends": ["./common.json"]
+}
+```
+
+Les entrées définies dans `extends` sont résolues à partir du chemin où le fichier `biome.json` est défini.
+Elles sont traitées dans leur ordre d’apparition, les réglages dans les derniers fichiers écrasant ceux dans les premiers.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:132]:
```suggestion
Elles sont traitées dans leur ordre d’apparition, les réglages d'une entrée écrasent ceux des entrées précédentes.
```

**Review by Conaclos — COMMENTED** [review]
Here is my first batch of suggestions. I didn't review the remaining part yet.

File: src/content/docs/fr/guides/configure-biome.mdx (hunk 6 above)
Comment (webdevbynight) [reply src/content/docs/fr/guides/configure-biome.mdx:20]:
```suggestion
Il est habituellement placé dans le répertoire racine de votre projet, où se trouve généralement `package.json`.
```

File: src/content/docs/fr/guides/configure-biome.mdx (hunk 11 above)
Comment (webdevbynight) [reply src/content/docs/fr/guides/configure-biome.mdx:79]:
```suggestion
Biome prend en compte le fichier de configuration le plus proche, en débutant sa recherche dans le répertoire de travail courant et en remontant la hiérarchie des répertoires jusqu’à trouver un fichier `biome.json` ou `biome.jsonc`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 15]
```diff
@@ -0,0 +1,386 @@
...
+}
+```
+
+Les entrées définies dans `extends` sont résolues à partir du chemin où le fichier `biome.json` est défini.
+Elles sont traitées dans leur ordre d’apparition, les réglages d’une entrée écrasant ceux des entrées précédentes.
+
+Biome est capable de résoudre des fichiers de configuration à partir du répertoire `node_modules/`.
+Donc, vous pouvez exporter votre fichier de configuration depuis un paquet et l’importer dans plusieurs projets.
+
+Pour ce faire, la première chose à faire est de mettre en place votre configuration de Biome «&nbsp;partagée&nbsp;» d’une certaine manière.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:136]: Je pense que nous pouvons supprimer cette phrase.
Elle n'apporte pas grand chose.
```suggestion
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 16]
```diff
@@ -0,0 +1,386 @@
...
+
+Assurez-vous que `@org/shared-configs` est correctement installé dans votre projet et mettez à jour le fichier `biome.json` pour qu’il ressemble à l’extrait suivant&nbsp;:
+
+```json title="biome.json"
+{
+  "extends": ["@org/shared-configs/biome"]
+}
+```
+
+Biome tentera de **résoudre** votre librairie `@org/shared-configs/` depuis votre répertoire de travail.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:158]:
```suggestion
Biome tentera de **résoudre** votre paquet `@org/shared-configs/` depuis votre répertoire de travail.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 17]
```diff
@@ -0,0 +1,386 @@
...
+Le répertoire de travail est&nbsp;:
+
+- en utilisant l’interface en ligne de commande, le répertoire depuis lequel vous exécutez vos scripts.
+  Cela correspond habituellement à la localisation de votre fichier `package.json`&nbsp;;
+- en utilisant le LSP, le répertoire racine de votre projet.
+
+:::caution
+Pour éviter un changement radical avec la manière dont la résolution existante fonctionne, les chemins commençant avec un point `.` ou finissant en `.json` ou `.jsonc` **ne seront pas** résolus depuis `node_modules/`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:166]:
```suggestion
Les chemins commençant avec un point `.` ou finissant en `.json` ou `.jsonc` **ne seront pas** résolus depuis `node_modules/`.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 18]
```diff
@@ -0,0 +1,386 @@
...
+La première façon de contrôler quels fichiers et répertoires sont traités par Biome est de les lister en ligne de commande.
+Avec la commande suivante, nous ne formatons que `file1.js` et tous les fichiers du répertoire `src`.
+Les répertoires sont parcourus de manière récursive.
+
+```shell
+biome format file1.js src/
+```
+
+:::caution
+Les motifs glob utilisés en ligne de commande ne sont pas interprétés par Biome.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:183]:
```suggestion
Les globs utilisés en ligne de commande ne sont pas interprétés par Biome.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 19]
```diff
@@ -0,0 +1,386 @@
...
+Ils sont développés par votre shell.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:184]:
```suggestion
Ils sont évalués par votre shell.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 20]
```diff
@@ -0,0 +1,386 @@
...
+Certains shells ne prennent pas en charge le glob récursif `**`.
+:::
+
+Le fichier de configuration de Biome peut être utilisé pour affiner les fichiers à traiter.
+Vous pouvez lister explicitement les fichiers à traiter en utilisant `include` et ceux à ne pas traiter en utilisant `ignore`.
+`include` et `ignore` acceptent des motifs glob comme `src/**/*.js`.
+Voyez la [section correspondante](#glob-syntax-explained) pour connaître les syntaxes glob prises en charge.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:191]:
```suggestion
Referrez-vous à la [section correspondante](#la-syntaxe-glob-expliquée) pour connaître les globs supportés.
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 21]
```diff
@@ -0,0 +1,386 @@
...
+`include` s’applique toujours en premier avant de faire appliquer `ignore`,
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:192]:
```suggestion
`include` s’applique toujours avant `ignore`,
```

File: src/content/docs/fr/guides/configure-biome.mdx [hunk 22]
```diff
@@ -0,0 +1,386 @@
...
+ce qui vous permet d’inclure certains fichiers et d’en ignorer certains parmi ceux que vous avez inclus.
+
+:::note
+`include` et `ignore` ont une sémantique légèrement différente.
+`include` n’empêche pas Biome de parcourir un dossier,
+ce qui veut dire que si vous voulez empêcher Biome de parcourir un dossier, vous devez ajouter le dossier à `ignore`.
```
Comment (Conaclos) [inline src/content/docs/fr/guides/configure-biome.mdx:198]:
```suggestion
`include` n’empêche pas Biome de parcourir un dossier.
Si vous voulez empêcher Biome de parcourir un dossier, vous devez ajouter le dossier à `ignore`.
```

File: src/content/docs/fr/guides/configure-biome.mdx (hunk 20 above)
Comment (webdevbynight) [reply src/content/docs/fr/guides/configure-biome.mdx:190]:
```suggestion
Référez-vous à la [section correspondante](#la-syntaxe-glob-expliquée) pour connaître les glob pris en charge.
```

**Comment by Conaclos** [comment]
Fantastic! And one more translated page :)

**Pull request merged**